homeassistant/components/natural_resources_wales/river_levels_sensor.py

```diff
@@ -0,0 +1,153 @@
+"""Classes for Natural Resources Wales river levels."""
```

Comment:
This should be extracted to a separate package published on PyPi, see:

https://developers.home-assistant.io/docs/creating_component_code_review/#4-communication-with-devicesservices